datadog_checks_downloader/tests/test_downloader.py

```diff
@@ -37,39 +64,289 @@
 ]
 
 
+@pytest.fixture(autouse=True)
+def _clear_cache():
+    """Clear the downloader's cache to make sure it does not affect test results."""
+    # current
+    metadata_current_dir = os.path.join(
+        os.path.dirname(datadog_checks.downloader.__file__),
+        "data",
+        "repo",
+        "metadata",
+        "current",
+    )
+    for file_ in os.listdir(metadata_current_dir):
+        if file_ == "root.json" or file_.startswith("."):
```

Comment:
Just curious, why do you use `file_` instead of `file` here?

Reply:
Adjusted in 05bf43c. It's my bad habit (I usually name `dir_` and `file_` because of `dir` built-in :))